.github/workflows/build-and-deploy.yml

```diff
@@ -0,0 +1,74 @@
+name: Build and Deploy FCA
+
+on:
+  pull_request:
+    branches:
+      - dev
```

Comment:
The reason I decided against is because of exactly what you said. The build that would be created would be the same as edge. No testing would be done with the pr build that would be created.